Tests/BacktraceBreadcrumbTests.swift

```diff
@@ -192,21 +188,10 @@ final class BacktraceBreadcrumbTests: QuickSpec {
                             }
                         }
 
-                        // Why the - 1?
-                        // Because one line is liable to get mangled by the wrapping - half will
-                        // be at the bottom and half at the top of the circular file.
-                        // Currently, we accept we lose this Breadcrumb in the UI - it will still be in the file
-                        // for manual inspection.
-                        let expectedNumberOfMatches = writeIndex - wrapIndex - 1
-                        expect(matches).to(beGreaterThanOrEqualTo(expectedNumberOfMatches),
-                                           description: "Not enough (\(matches) out of \(expectedNumberOfMatches)) " +
-                                           "breadcrumb matches found in breadcrumbs file: \n\(breadcrumbText)")
-
                         let attr = try FileManager.default.attributesOfItem(atPath: self.breadcrumbLogPath(false))
                         let fileSize = attr[FileAttributeKey.size] as? Int
                         let requestedSize = settings.maxQueueFileSizeBytes
                         expect { fileSize }.to(beLessThanOrEqualTo(requestedSize))
-                        expect { fileSize }.to(beGreaterThanOrEqualTo(requestedSize - 1000))
```

Comment:
This check is important. I would consider adjusting it, not deleting it.

Reply:
will do